cli/Cargo.toml

```diff
@@ -74,13 +74,15 @@ experimental = [
     # The following features are experimental:
     "circuit-auth-type",
     "health",
+    "registry-add-node",
     "splinter-cli-jwt",
 ]
 
 circuit-auth-type = []
 circuit-template = ["splinter/circuit-template"]
 
 splinter-cli-jwt = ["cylinder/jwt", "cylinder/key-load"]
+registry-add-node=[]
```

Comment:
Maybe we can just do `registry` and put all the experimental registry commands under this feature instead of having a separate feature for each; that way we can stabilize them all at once. Also, make sure this is formatted consistently (spaces around the `=`).

Reply:
changed to just `registry`